### DISCUSSION TITLE
Auto-lock closed issues

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#12575] [General]
Bokeh is one of the few large projects I have worked on that doesn't have an auto-lock process of some sort at this point. At a minimum, I would like to propose auto-locking old issues and PRs that are closed and resolved. There is no reason for anyone to necro-bump these (just got one on six year old issue). The lock message can direct users to open a new issue, with a link to the old one of that is really necessary. 

cc @bokeh/core

**Comment by ianthomas23** [comment]
:+1: on this.

**Comment by mattpap** [comment]
> (...) old issues and PRs that are closed and resolved. 

With the emphasis on resolved, because otherwise I view this a dismissive towards submitters. Though, I would prefer to have a resolution (`reso:*` tag) on all closed issues. I've seen a recent influx of issues being closed without a clear resolution and possibly marked as `type: discussion`.

**Reply by bryevdv** [reply]
Clarifying that by "are resolved" I mean "has a `reso:` tag"

>  I would prefer to have a resolution (reso:* tag) on all closed issues. 

There's no clear line for when something is a GH dicussion vs `type: discussion` (just because the latter predates the former) or if we should get rid of one in favor of the other. But that is a separate topic. I'd suggest opening a separate discussion for this.

**Comment by bryevdv** [comment]
It seems like there is some interest in this, so I will at least start to investigation what specific technical options are available.
